docs/src/finfield.md

```diff
@@ -41,6 +41,11 @@ julia> g = F(3)
 julia> h = F(g)
 3
 
+julia> GF(4)
+ERROR: DomainError with 4:
+Characteristic is not prime in GF(p)
+Stacktrace:
+[...]
```

Comment:
Are you sure this will pass? It's in the jldoctests section. The output needs to be precisely what Julia outputs, I thought.

Reply:
This is a supported feature (cf. https://docs.julialang.org/en/v1.4-dev/manual/documentation/), it allows to skip irrelevant details.